omero/developers/Web/ViewPort.txt

```diff
@@ -40,6 +41,22 @@ Then in ``<BODY>`` insert the following:
 
         <a href="#" onclick="return openPopup('http://localhost:8000/webclient/img_detail/IMAGE_ID/')">Open viewer</a>
 
+
+Example: 
+^^^^^^^^
+
+.. raw:: html
+
+    <p style="width:160px; margin-right:10px; padding:5px; background-color: #f8f8f8; border: 1px solid #ccc;">
+        <a onclick="window.open(this.href, '', 'resizable=no,status=no,location=no,width=800,top=100,left=100,height=600,toolbar=no,menubar=no,fullscreen=no,scrollbars=no,dependent=no'); return false;" href="https://nightshade.openmicroscopy.org/webgateway/img_detail/3933597/">
+             <img src="https://nightshade.openmicroscopy.org/webgateway/render_thumbnail/3933597/150/150/" style="margin:5px;"/>
+        </a>
+        <i>Click on the thumbnail to view and manipulate the image in OMERO.</i>
```

Comment:
That is expected, because nightshade is broken. After 5.1.1 upgrade it will come back to normal, see https://github.com/openmicroscopy/openmicroscopy/pull/3707